mailscanner/functions.php

```diff
@@ -287,6 +286,12 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
         }
     }
 
+    if (!isset($_SESSION['last_update']) || $_SESSION['last_update'] + SESSION_TIMEOUT < time()) {
```

Comment:
This would always be true because the last_update has already been overwritten in line 127